packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -47,23 +46,23 @@ class EnrollVerbHandler extends AbstractVerbHandler {
       throw UnAuthenticatedException(
           'Cannot $operation enrollment without authentication');
     }
+    EnrollParams? enrollVerbParams;
     try {
-      var enrollVerbParams;
       if (verbParams[enrollParams] != null) {
```

Comment:
if verbParams[enrollParams] is null we have to throw exception and exit..rest of the code will not work without enrollParams

Reply:
enroll:list does not have enrollParams. Have included a check in processVerb to ensure that all operations except enroll:list have valid enrollParams. Change present in [4ef1efb](https://github.com/atsign-foundation/at_server/pull/1536/commits/4ef1efb0c490d5ec2459a5d7768fa917a0b28ab8)